test/files/rules/novarkeyword.test.ts

```diff
@@ -0,0 +1,25 @@
+
+// Error
+var foo;
+
+function tmp(t: any) {
+    var x = 3;
+}
+
+var i,
+    j;
+
+
+// Valid
+declare var tmp: any;
+
+
+let bar;
+const qux;
+
+let k,
+    l;
+
+module quz {
+    export var i;
+}
```

Comment:
let's add some more tests for 1.5 syntax, like `export default`, `import`, destructuring (`var [a, b] = [1, 2]`)...